indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -0,0 +1,58 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+
+{% macro _render_regform_item_col_selector(item) %}
+    {% if item.is_section %}
+        <h3>{{ item.title }}</h3>
+        {% for child_item in item.children %}
+            {% if child_item.is_field %}
+                {% set has_dropdown = child_item.input_type in ['radio', 'country'] %}
+                <div class="i-label reglist-column js-regform-item {%- if has_dropdown %} filter{% endif %}"
+                     data-id="{{ child_item.id }}">
+                    <span class="title {% if has_dropdown %}icon-filter{% endif %}"
+                          data-toggle="dropdown">
+                        {{- child_item.title -}}
+                    </span>
+                    {% if has_dropdown %}
+                        <ul class="dropdown" style="display: none">
+                            {% for radio_item in child_item.current_data.versioned_data.radioitems %}
+                                <li>
+                                    <input type='checkbox' name='radiogroup_{{ child_item.id }}'
+                                           value='{{ radio_item.id }}'
+                                           id="{{ radio_item.id }}"
+                                           {% if radio_item.id in filters %}
+                                               checked
+                                           {% endif %}>
+                                    <label for="{{ radio_item.id }}">
+                                        {{ radio_item.caption }}
+                                    </label>
+                                </li>
+                            {% endfor %}
+                        </ul>
+                    {% endif %}
+                    <span class="right actions">
+                        <span class="trigger icon-eye"></span>
+                    </span>
+                </div>
+            {% endif %}
+        {% endfor %}
+    {% endif %}
+{% endmacro %}
+
+{% block content -%}
+    <form class="reglist-filter" method="POST">
+        {% set visible_cols_regform_items = visible_cols_regform_items | tojson | forceescape %}
+        <input id="visible-columns-reg-items" type="hidden" name="visible_cols_regform_items"
+               value="{{ visible_cols_regform_items }}">
+        {% for item in regform.form_items %}
+            {{ _render_regform_item_col_selector(item) }}
+        {% endfor %}
+        <div class="bottom-buttons">
+            <input class="i-button big highlight" type="submit" data-disabled-until-change
```

Comment:
What about a button to clear all filters? Could simplt be a submit button with `name="reset'` and then you check `if request.form.get('reset')` in the RH